objects/ogarbagecan/Create_0.gml

```diff
@@ -0,0 +1,36 @@
+/// @description Insert description here
+// You can write your code in this editor
+
+event_inherited();
+
+hp = 30;
+
+knockbackMultiplier = 1.5; // Multiplier for how much knockback this object takes.
+launchable = true; // Whether this object can be launched or not
+isThrowable = true; // whether we can throw this object around or not
+
+hasWallCollision = false;
+
+sprite_index = sGarbageCan;
+
+hitstun = 0;
+inAttackState = false;
+
+with (hurtbox) 
+{
+	primary = true;
+	owner = other.id;
+	image_xscale = 21;
+	image_yscale = 32;
+}
+hurtboxOffset = -11;
+
+CharacterSprites = {
+	idle_Sprite : sprite_index,
```

Comment:
Same here, pretty sure these variable names are inherited, but if not use camelCase and no underscores, like ```hurtSprite``` etc.

Reply:
The code for hitboxes checks the same variables in the destructible objects as it does in the players, so the variables must be the same name.